client/share/freeipa.template

```diff
@@ -1,2 +1,12 @@
 [libdefaults]
     spake_preauth_groups = edwards25519
+
+[plugins]
+    kdcpolicy = {
+        module = ipakdb:kdb/ipadb.so
+        enable_only = ipakdb
+    }
+    certauth = {
+        module = ipakdb:kdb/ipadb.so
+        enable_only = ipakdb
+    }
```

Comment:
Isn't ipadb.so only shipped in servers now?

Reply:
Yes, ``ipadb.so`` is only available in ``ipa-server`` package. Please move the snippet to a server specific template.